app-backend/api/src/main/resources/application.conf

```diff
@@ -50,6 +50,9 @@ auth0 {
 }
 
 s3 {
+  region = "us-east-1"
+  region = ${?AWS_DEFAULT_REGION}
```

Comment:
It appears as though this is supposed to be `AWS_REGION`, but unclear which version of the SDK we're actually using to be 100% certain: https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/SDKGlobalConfiguration.java#L216